common/thorhelper/thorread.cpp

```diff
@@ -1303,6 +1303,7 @@ const void *MarkupDiskRowReader::nextRow(size32_t & resultSize)
 {
     const void * next = nextRow(bufferBuilder);
     resultSize = tempOutputBuffer.length();
+    tempOutputBuffer.clear();
```

Comment:
logically this should be called before the call to nextRow(), rather than after.  Theoretically clear() could destroy the contents of the buffer.

Reply:
Fixed.